client/src/modules/dashboard/Events/components/EventFormUtils.ts

```diff
@@ -132,7 +132,8 @@ export interface EventFormData {
 
 export type IEventData = Pick<
   Event,
-  keyof Omit<EventFormData, 'venue_id' | 'tags' | 'sponsors'> | 'id'
+  | keyof Omit<EventFormData, 'venue_id' | 'tags' | 'sponsors' | 'chapter_id'>
+  | 'id'
```

Comment:
```suggestion
  keyof Omit<EventFormData, 'venue_id' | 'tags' | 'sponsors'> | 'id'
```
Nit: `chapter_id` isn't in `EventFormData`